json/json_test.go

```diff
@@ -18,6 +19,8 @@ import (
 
 var ErrResponseError = errors.New("response error")
 
+var beforeAfterContext = map[string]string{}
```

Comment:
can you make this a member of the Service1 type instead? I'd like to avoid any kind of global state in tests